flutter-for-react-native.md

```diff
@@ -1873,7 +1863,7 @@ class _LogoFadeState extends State<LogoFade> with TickerProviderStateMixin {
 
 ### How do I add swipe animation to Cards?
 
-In React Native, you either use PanResponder or third-party libraries for swipe animation. In Flutter, we can achieve this animation by using the [`Dismissible`](https://docs.flutter.io/flutter/widgets/Dismissible-class.html) widget and nest the child widgets.
+In React Native, you either use PanResponder or third-party libraries for swipe animation. In Flutter, we can achieve this animation simply by using the [`Dismissible`](https://docs.flutter.io/flutter/widgets/Dismissible-class.html) widget and nest the child widgets.
```

Comment:
we should be wary of adding self-congratulating adjectives like this. Let our audience determine if it's simple or not.

For example, using Dismissible isn't actually all that simple; you have to use a stateful widget, track callbacks, update lists, regenerate children, etc. So people might read this, then get upset that we misled them about how simple it was.